src/screens/Onboard/UserOnBoard.css

```diff
@@ -0,0 +1,21 @@
+.onboard{
+    display:flex;
+    justify-content:center;
+    align-items:center;
```

Comment:
You can create a utility class for all these three flex properties, you'll need them many times in the ui